src/entities/exchangeRates.ts

```diff
@@ -117,6 +120,15 @@ export function updateExchangeRates(exchangeRate: ExchangeRate, timestamp: BigIn
     }
   }
 
+  const ssvAssetUniPool = Address.fromString(SSV_ASSET_UNI_POOL)
+  if (ssvAssetUniPool.notEqual(Address.zero())) {
+    const pool = UniswapPool.load(ssvAssetUniPool.toHex())
```

Comment:
It will always return null as we don't store this pool. Please check `isSupportedToken` and how it is used.
We must update it so that we don't store uniswap positions for this pool, but store the pool itself and it's latest state.